riverdriver/riverdrivertest/driver_test.go

```diff
@@ -111,7 +112,59 @@ func TestDriverRiverPgxV5(t *testing.T) {
 		})
 }
 
-func TestDriverRiverSQLite(t *testing.T) {
+func TestDriverRiverLiteLibSQL(t *testing.T) { //nolint:dupl
```

Comment:
Is it worth setting these up to use the same test logic to avoid having to add the `nolint:dupl`? Or are there subtle differences between the two?